scripts/A16.R

```diff
@@ -0,0 +1,76 @@
+# create ds object --------------------------------------------------------
+
+ds <- create_dataset(id = "A16")
+
+
+# download the data -------------------------------------------------------
+
+ds <- download_data(ds)
+
+# data cleaning -----------------------------------------------------------
+
+### Filter specific dimensions
+
+# To reduce the dataset size, we will not include sex and citizenship category
+ds$data %>%
+  janitor::clean_names() %>%
+  dplyr::filter(
+    sex == "Sex - total" &
+      citizenship_category == "Citizenship (category) - total"
```

Comment:
I think I would leave the citizenship category in, since it relates to the other columns, where as gender does not: 

For example the out migration and in migration: it is interesting whether Swiss citizens left the country or came into it from another country or whether it was other nationalities that left or came.